README.md

```diff
@@ -60,11 +60,117 @@ You can see a big picture of the roadmap here: [**ROADMAP**][roadmap]
 - [Wallet types](https://github.com/kommitters/kadena.ex/issues/18)
 - [Kadena Crypto](https://github.com/kommitters/kadena.ex/issues/51)
 - [Kadena Pact](https://github.com/kommitters/kadena.ex/issues/55)
+- [Kadena Pact Commands Builder](https://github.com/kommitters/kadena.ex/issues/131)
 
 </details>
 
 ---
 
+## Building Commands
+
+### Command
+
+It allows construction of PACT command payloads, in a composable and semantic manner, which will be part of the body of a request to the pact api endpoints, the commands which allows to build are [Execution](#execution-command) command and `Continuation` command
+
+### [Execution Command](#execution-command)
+
+An execution command is the first contract state that contains the actor keys and the code that will be run once the contract's requirements have been met.
+To create an execution command is needed:
+
+- [Network Id](#network-id)
+- Codigo (PACT)
+- Nonce
+- [Enviroment Data](#env-data)
+- [Meta Data](#meta-data)
+- [Key Pairs](#key-pair)
+- [Signers](#signers-list)
+
+### [Network ID](#network-id)
```

Comment:
@Odraxs @EdwinGuayacan Network ID, Env Data, Meta Data, KeyPairs, and Signers should be nested into `Execution Command`, so please use 4 sharps instead of 3: `#### [Network ID](#network-id)`